installers/unix.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+zip_url=https://github.com/tomasklaen/uosc/releases/latest/download/uosc.zip
+conf_url=https://github.com/tomasklaen/uosc/releases/latest/download/uosc.conf
+zip_file=/tmp/uosc.zip
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+cleanup() {
+	if [ -f "$zip_file" ]; then
+		echo "Deleting: $zip_file"
+		rm -f $zip_file
+	fi
+}
+
+abort() {
+	cleanup
+	printf "%s\n" "$@" >&2
+	exit 1
+}
+
+# Check OS
+OS="$(uname)"
+if [ "${OS}" == "Linux" ]; then
+	data_dir="${XDG_CONFIG_HOME:-~/.config}/mpv"
+elif [ "${OS}" == "Darwin" ]; then
+	data_dir=~/Library/Preferences/mpv
+else
+	abort "This install script works only on linux and macOS."
+fi
+
+# Remove old and deprecated folders & files
+echo "Deleting old and deprecated uosc files and directories."
+rm -rf "$data_dir/scripts/uosc_shared"
+rm -rf "$data_dir/scripts/uosc"
+rm -f "$data_dir/scripts/uosc.lua"
+
+# Install new version
+echo "Downloading: $zip_url"
+curl -o $zip_file $zip_url
```

Comment:
```suggestion
curl -L -o $zip_file $zip_url
```
curl doesn't deal with the redirect unless `-L` is passed